_posts/2017-08-31-jsonp.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: "JSONP: what and how"
+date: 2017-08-31 09:35:16
+description: Qué es Jsonp y cómo usarlo.
+tags: Angular Jsonp API
+categories: Frontend API
+twitter_text: JSONP: what and how
```

Comment:
Please, pon "JSONP: what and how" entre comillas, el analizador de texto se raya al ver otros dos puntos en la linea.